terraform.tfvars.example

```diff
@@ -33,6 +33,23 @@ gpuNodeSize = [
   "p3.2xlarge"
 ]
 
+# The minimum number of nodes for IVS gpu job execution
+ivsGpuNodeCountMin = 0
+
+# The maximum number of nodes for IVS gpu job execution
+ivsGpuNodeCountMax = 2
+
+# The disk size in GiB of the nodes for the IVS gpu job execution
+ivsGpuNodeDiskSize = 100
+
+# Specifies whether an additional node pool for IVS gpu job execution is added to the kubernetes cluster
+ivsGpuNodePool = false
```

Comment:
I prefer to have this variable first of ivs options since it is enable/disable addition of nodepool.

Reply:
Done